modules/apps/content-dashboard/content-dashboard-web/src/main/resources/content/Language.properties

```diff
@@ -17,7 +17,7 @@ in-use=In Use
 javax.portlet.title.com_liferay_content_dashboard_web_portlet_ContentDashboardAdminPortlet=Content Dashboard
 languages-translated-into=Languages Translated Into
 metrics=Metrics
-no-x-category=No {0} Category
+no-x-specified=No {0} Specified
```

Comment:
let's see if brian likes approves this statement

Reply:
Agree. I don't know...